openmdao/visualization/opt_report/tests/test_opt_report.py

```diff
@@ -163,8 +163,14 @@ def test_opt_report_run_once_driver(self):
                                           )
         expect = {'obj_calls': 0, 'deriv_calls': 0}
         self.check_opt_result(expected=expect)
-        opt_report(self.prob)
-        self.check_opt_report(expected=expect)
+
+        expected_warning_msg = "The optimizer report is not applicable for the Driver Driver " \
```

Comment:
Tangential issue, but we probably want to put the driver name in single quotes in this error message, so that it looks like `for the Driver 'Driver'`.

Reply:
Sorry meant to make this fix before merging... adding it to #2716